File: Tarefa/Funcionarios.java
public class Funcionarios {

	private String nome;
	private String CPF;
	private double salario_base;
	private double cargo;

	/**
	 * 1:vendedor; 2:assistente; 3:gerente
	 **/

	public Funcionarios(String name, int num) {
		nome = name;
	}

	public String getNome() {
		return nome;
	}

	public void setNomeCliente(String nome) {
		this.nome = nome;
	}

	public String getCPF() {
		return CPF;
	}

	public void setCPF(String CPF) {
		this.CPF = CPF;
	}

	public double getSalario_base() {
		return salario_base;
	}

	public void setSalario_base(double s) {
		this.salario_base = s;
	}

	public double getCargo() {
		return cargo;
	}

	public void setCargo(double c) {
		this.cargo = c;
	}
}

File: Tarefa/Funcionarios.py
class Funcionarios:
    def __init__(self, n, num, sal):
        name =""
        self.nome = name
        self.CPF = ""
        self.salario_base = 0.0
        self.cargo = 0.0

    def getNome(self):
        return self.nome

    def setNomeCliente(self, nome):
        self.nome = nome

    def getCPF(self):
        return self.CPF

    def setCPF(self, CPF):
        self.CPF = CPF

    def getSalario_base(self):
        return self.salario_base

    def setSalario_base(self, s):
        self.salario_base = s

    def getCargo(self):
        return self.cargo

    def setCargo(self, c):
        self.cargo = c

